src/current/v22.2/install-cockroachdb-mac.md

```diff
@@ -52,9 +52,22 @@ Use one of the options below to install CockroachDB.
         {% include marketo-install.html uid="1" %}
     </li>
   </ol>
-{{site.data.alerts.callout_info}}
-If you previously installed CockroachDB via Homebrew, run <code>brew uninstall cockroach</code> before installing the new version. If you installed using a different method, you may need to remove the binary before installing via Homebrew.
-{{site.data.alerts.end}}
+{{site.data.alerts.callout_success}}
+If you previously installed CockroachDB via Homebrew, you can [upgrade]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}) the CockroachDB binary to the next major version or to a patch version using HomeBrew. After updating the binary on each node, restart the `cockroach` process on the node. When upgrading to a new major version, you must complete additional steps to [finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#step-6-finish-the-upgrade) the upgrade. If you need to upgrade through multiple major versions, you must complete each major-version upgrade separately, including finalizing the upgrade, before beginning the next one.
```

Comment:
The links are not rendering in each of the versions. I'm not sure why? In v22.2, I think we want to use the old link format... however the rendering issue is also present in v23.1 & v23.2